xarray/backends/api.py

```diff
@@ -788,6 +789,33 @@ def open_dataarray(
     return data_array
 
 
+def open_datatree(
+    filename_or_obj: str | os.PathLike[Any] | BufferedIOBase | AbstractDataStore,
+    engine: T_Engine = None,
+    **kwargs,
+) -> DataTree:
+    """
+    Open and decode a dataset from a file or file-like object, creating one Tree node for each group in the file.
+    Parameters
+    ----------
+    filename_or_obj : str, Path, file-like, or DataStore
+        Strings and Path objects are interpreted as a path to a netCDF file or Zarr store.
+    engine : str, optional
+        Xarray backend engine to us. Valid options include `{"netcdf4", "h5netcdf", "zarr"}`.
+    kwargs :
+        Additional keyword arguments passed to :py:meth:`~xarray.open_dataset` for each group.
+    Returns
+    -------
+    xarray.core.datatree.DataTree
```

Comment:
The datatree class isn't migrated yet, so this is also incorrect for now

Reply:
I think you're saying we should just update the comments to where the code will eventually live?  I can get behind that.   I'm still uncertain of what [this](https://github.com/pydata/xarray/pull/8697/files/b74764eced9af005220c93351cce5b87d757c3d5#r1485671445) comment was.